### DISCUSSION TITLE
items-start consistency

### DISCUSSION THREAD

**Discussion by claudeG** [tailwindlabs/tailwindcss#10243] [Ideas]
Tailwind defines .items-start { align-items: flex-start }
There is 3 problems:

- this rule ("start") does not tell accurately what the added behaviour is (flex-start).
- As a user I may prefer { align-items: start } which behaves differently from { align-items: flex-start }
In such case, 1) this rule does not exist and 2) its creation obliges me to introduce a puzzling name
- As a user I am not happy to add a flex-oriented behaviour on a grid.

See also https://csslayout.news/whats-the-difference-between-the-alignment-values-of-start-flex-start-and-self-start/

**Comment by denial512** [comment]
Hello
Isolate one goal. Developing consistency goes against human nature. 
Focus on incremental improvement. You're not going to develop a positive, worthwhile habit overnight. 
Fight your emotions. The brain is a taxing organ.  [BenefitsCal](https://www.benefitscal.biz/)
Forgive your failures. That said, you are human, imperfect like the rest of us
